### PR TITLE
Update website to emphasize ability to play against community bots

### PR DESCRIPTION
**Pull request by azeemba**
- Updated the tag line to add sharing of bots
- Emphasized being able to play against community bots
- Moved discord link up because presumably we want everyone to join, not just botmakers

Here is a screenshot:

image

I did add a story mode shoutout, let me know if we think it doesn't fit.

### CODE REVIEW

**Comment by tarehart** [comment]
I like the new content! I think it got a little too long though. Here's my attempt to shorten it:


RLBot teaches programmers to create custom offline Rocket League bots! Even if you don't code, you can easily play with community bots or challenge yourself with our story mode.

Quick Start Video
RLBot for Windows or see our OS support

You can read more on our wiki, and join our Discord server to chat about the possibilities. If you're ready to start, click your favorite language below!

**Comment by azeemba** [comment]
I agree, I had a concern about length too.

Your suggestion looks great!

One thing I was trying to do was make it so that non-botmakers see the Discord server as something they should join as well. That's why I tried to separate it out from other "botmaker" specific things. Do you have thoughts on this? Maybe its not such a big deal?

**Comment by tarehart** [comment]
I agree that inviting non-botmakers into the discord is desirable, not sure yet how to improve the wording.

**Comment by nevercast** [comment]
> You can read more on our wiki, and join our Discord server to chat about the possibilities. If you're ready to start, click your favorite language below!

Maybe reworded to:

> If you're interested in participating in our community, join our Discord server to chat about the possibilities. If you're ready to get started, you can read more on our Wiki or click your favourite language below!

Intent being to separate "Discord" from "Wiki and Languages" as a whole Community and Developers thing.

**Comment by tarehart** [comment]
Good suggestion!

> RLBot teaches programmers to create custom offline Rocket League bots! Even if you don't code, you can easily play with community bots or challenge yourself with our story mode.
> 
> Quick Start Video
> RLBot for Windows or see our OS support
> 
> Join our Discord chat server to participate in the community. If you're ready to get started, you can read more on our Wiki or click your favourite language below!

How's that?

**Comment by nevercast** [comment]
LGTM! :tada:

**Comment by azeemba** [comment]
Thanks for the suggestions!

image

**Comment by ddthj** [comment]
RLBot teaches programmers? non programmers can easily play? What are they playing if RLBot is for teaching?

RLBot enables custom bots in Rocket League. Using the RLBotGUI you can download and play with community-made bots, play our challenging RLBot story mode, or make your own bot!

**Comment by tarehart** [comment]
I think what we've got is a big improvement over what's currently on rlbot.org, I'm gonna go ahead and merge. Goose let's pick up that conversation in a followup pull request if you're up for it.

**Comment by nevercast** [comment]
:clap: Hazzah on the merge.
